client/homepage/stats-overview/index.js

```diff
@@ -89,7 +104,47 @@ export const StatsOverview = ( { userPrefs, updateCurrentUserData } ) => {
 				/>
 			}
 		>
-			Content Here
+			<TabPanel
+				className="woocommerce-stats-overview__tabs"
+				tabs={ [
+					{
+						title: __( 'Today', 'woocommerce-admin' ),
+						name: 'today',
+					},
+					{
+						title: __( 'Week to date', 'woocommerce-admin' ),
+						name: 'week',
+					},
+					{
+						title: __( 'Month to date', 'woocommerce-admin' ),
+						name: 'month',
+					},
+				] }
+			>
+				{ ( tab ) => (
+					<ul className={ listClasses }>
+						<StatsList
+							query={ {
+								period: tab.name,
```

Comment:
Are we going to introduce a new param `period` for the performance indicators?  While I think this would be nice, I went with the presumption that we would fit performance indicators under the existing `before` and `after` filters that we currently use in https://github.com/woocommerce/woocommerce-admin/pull/4291.

I think it reduces the API complexity a bit if we can keep this params as consistent as possible, but definitely open for discussion.

Reply:
Right, so the other component that displays performance indicators is StorePerformance, which uses the `query` object and translates the `period` property into `before` and `after`. Here, I'm keeping consistent with that input even though its not derived from the url query. 

>I think it reduces the API complexity a bit if we can keep this params as consistent as possible, but definitely open for discussion.

As far as the API is concerned, `before` and `after` are still being used as is. StatsList converts the `period` to `before` and `after` using the re-used logic from StorePerformance.

Is that satisfactory?